### ISSUE TITLE
Not submitting orders

### ISSUE THREAD

**Issue by damanijbrown** [ranaroussi/qtpylib#63]
Testing in Papertrade and Backtest, noticed position always stayed at 0. 

I even did a "market order" keeps saying I have pending orders. So, I check my paper trade account, and of course no pending trades. What's up with that.

Sol, I decided to print positions and this is what it prints.

{'symbol': 'AAPL', 'position': 0, 'avgCost': 0.0, 'account': None}

Account says "None" (I do have multiple accounts in my main, which also returns an error upon blotter connecting in live. Is there some sort of flag that I need to add to set the account? is that why it's not submitting positions?

**Comment by damanijbrown** [comment]
2017-04-06 12:14:17,442 [WARNING] qtpylib.broker: Not submitting AAPL order, orders pending: {'AAPL': {'orderId': 1, 'quantity': 1, 'expires': datetime.datetime(2017, 4, 6, 12, 15, 12, 30058)}}

**Comment by ranaroussi** [comment]
Are you using the latest version? This was supposed to have been fixed in the last version...

Please run this and try your code again:
```bash
$ pip install qtpylib --upgrade --no-cache-dir
```

**Comment by damanijbrown** [comment]
I have 1.5.71 and all dependancies are up to date.

**Comment by damanijbrown** [comment]
Just for a test 

 def on_tick(self, instrument):
        # optional method that gets called on every tick received
        # optional method that gets called on every bar received

        # buy if position = 0, sell if in position > 0
        if instrument.positions['position'] == 0:
            instrument.buy(100)
            print("Buying shares")
            print(instrument.positions['position'])
        else:
            instrument.exit()

---------
Buying shares
2017-04-18 11:13:51,168 [WARNING] qtpylib.broker: Not submitting AAPL order, orders pending: {'AAPL': {'orderId': 1, 'quantity': 100, 'expires': datetime.datetime(2017, 4, 18, 11, 14, 27, 355938)}}
0
2017-04-18 11:13:51,367 [WARNING] qtpylib.broker: Not submitting AAPL order, orders pending: {'AAPL': {'orderId': 1, 'quantity': 100, 'expires': datetime.datetime(2017, 4, 18, 11, 14, 27, 355938)}}
Buying shares
0
Buying shares
2017-04-18 11:13:51,868 [WARNING] qtpylib.broker: Not submitting AAPL order, orders pending: {'AAPL': {'orderId': 1, 'quantity': 100, 'expires': datetime.datetime(2017, 4, 18, 11, 14, 27, 355938)}}
0
2017-04-18 11:13:52,119 [WARNING] qtpylib.broker: Not submitting AAPL order, orders pending: {'AAPL': {'orderId': 1, 'quantity': 100, 'expires': datetime.datetime(2017, 4, 18, 11, 14, 27, 355938)}}
Buying shares
0
2017-04-18 11:13:52,371 [WARNING] qtpylib.broker: Not submitting AAPL order, orders pending: {'AAPL': {'orderId': 1, 'quantity': 100, 'expires': datetime.datetime(2017, 4, 18, 11, 14, 27, 355938)}}
Buying shares
0
2017-04-18 11:13:53,483 [WARNING] qtpylib.broker: Not submitting AAPL order, orders pending: {'AAPL': {'orderId': 1, 'quantity': 100, 'expires': datetime.datetime(2017, 4, 18, 11, 14, 27, 355938)}}
Buying shares
0

**Comment by ranaroussi** [comment]
Should be fixed in latest release (v 1.5.72).

Please upgrade using:

```bash
$ pip install qtpylib --upgrade --no-cache-dir
```

...and let me know if that works now.

**Comment by ranaroussi** [comment]
Closing this for now. Please re-open if this issue resurfaces.
